docs/FreeBSD.md

```diff
@@ -123,53 +123,12 @@ Thanks for using Minio, awaiting feedback :-)
 
 It is possible to build the minio server from source on FreeBSD.  To do this we will used the golang distribution provided by the FreeBSD pkg infrastructure.
 
-First we will need to install golang as well as GNU make:
+First we will need to install golang, follow [https://docs.minio.io/docs/how-to-install-golang](how to install golang) guide. We also need to install GNU make:
```

Comment:
I agree, Golang as well as gnu-make are both available in FreeBSD ports and pkg system.  To install golang via pkg you should be able to simply run:

$ sudo pkg install go

This will make it easier for end-users to keep up to date in terms of bugfixes and new releases as opposed to installing from source directly from Golang project.